app/views/devise/registrations/edit.html.erb

```diff
@@ -49,6 +49,11 @@
         <%= f.select :gender, [["Masculino", "male"], ["Femenino", "female"]] %>
       </div>
 
+      <div class="field">
+          <%= f.label :role, 'Usuario' %>
+          <%= f.select :role, [["Huesped", "guest"], ["Anfitrión", "host"]], {}, onchange: "showDni();" %>
+        </div>
```

Comment:
the identation could be 2 spaces, please fix it!

Reply:
Fixed!